src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -196,6 +266,29 @@ We refer to the common principles for the standards for describing dates and tim
 
```

Comment:
I can't edit this directly, but above this line, the sentence 

> We refer to the common principles for the standards for describing dates and timestamps, including possibilities for anonymization (see the [units section](../02-common-principles.md#units)).

should be changed so that the final link says: `(see [Units](../02-common-principles.md#units))`

Reply:
Thanks. Addressed in c13df8d.